doc/classes/GPUParticles2D.xml

```diff
@@ -132,6 +132,7 @@
 			Particles are drawn in order of remaining lifetime.
 		</constant>
 		<constant name="DRAW_ORDER_REVERSE_LIFETIME" value="2" enum="DrawOrder">
+			Particles are drawn in reverse order of remaining lifetime.
```

Comment:
I would add a concrete example for this (also for Lifetime). Does this mean the newest particles are drawn in front, or the oldest particles?

Reply:
Made the changes. Not sure if I worded it right. Do check if they are fine.